watsononlinestore/watson_online_store.py

```diff
@@ -276,7 +278,7 @@ def get_discovery_response(self, input_text):
         # Format response to show user.
         formatted_response = ""
         for item in response:
-            formatted_response += "\n" + str(item['item']) + ") " + \
+            formatted_response += "\n" + str(item['cart_number']) + ") " + \
```

Comment:
OK. I just changed the name from 'item' to 'cart_number' to make it more clear. Your commit was the one that originally cast item['item'] as a string...